crates/wasi-common/Cargo.toml

```diff
@@ -93,6 +93,11 @@ tokio = [
 ]
 exit = [ "wasmtime", "dep:libc" ]
 
+[package.metadata.docs.rs]
+features = ["trace_log", "wiggle_metadata", "wasmtime", "sync", "tokio", "exit"]
+# Define the configuration attribute `docsrs`. Used to enable `doc_cfg` feature.
+rustdoc-args = ["--cfg", "docsrs"]
```

Comment:
I think that this is the default on docs.rs, so I think it's ok to leave this out (but still use `cfg_attr(docrs, ...)` in the code

Reply:
Done